### PR TITLE
Multilang: add missing Chinese (Simplified) translations

### PR DESCRIPTION
**Pull request by vcerchou**

### CODE REVIEW

File: selfdrive/ui/translations/main_zh-CHS.ts [hunk 1]
```diff
@@ -731,7 +731,7 @@ location set</source>
     <name>Sidebar</name>
     <message>
         <source>CONNECT</source>
-        <translation>CONNECT</translation>
+        <translation>连接</translation>
```
Comment (sshane) [inline selfdrive/ui/translations/main_zh-CHS.ts:734]: Connect is the branding for a product, I'd leave it in English

**Comment by vcerchou** [comment]
这是一封自动回复邮件。已经收到您的来信，我会尽快回复。

File: selfdrive/ui/translations/main_zh-CHS.ts [hunk 2]
```diff
@@ -124,7 +124,7 @@
     </message>
     <message>
         <source>N/A</source>
-        <translation>N/A</translation>
+        <translation>空</translation>
```
Comment (sshane) [inline selfdrive/ui/translations/main_zh-CHS.ts:127]: Can we keep this PR to only adding new translations?

Comment (vcerchou) [reply selfdrive/ui/translations/main_zh-CHS.ts]: OK

**Comment by sshane** [comment]
We thought the translations in https://github.com/commaai/openpilot/pull/26492 were a bit more natural. Feel free to fix any typos or improve translations in another PR.

**Pull request closed without merging**